sharedbuffers/mapped_struct.py

```diff
@@ -716,30 +788,30 @@ def pack_into(cls, obj, buf, offs, idmap = None, implicit_offs = 0,
             if isinstance(obj, npndarray):
                 a = obj
             else:
-                a = array(dtype, obj)
+                a = array(dtype.decode(), obj)
             abuf = buffer(a)
             buf[offs:offs+len(abuf)] = abuf
             offs += len(abuf)
-            offs = (offs + 7) / 8 * 8
+            offs = (offs + 7) // 8 * 8
```

Comment:
The best way to avoid headaches with division semantics in dual code, is to add `from __future__ import division` to the imports on the top, and make sure we always consistently use py3 semantics.

This means using `//` here as you did, but without the future import this will break py2.

Reply:
Ping. Add the future import.

Reply:
It's already there :P

Reply:
No... where?

Reply:
NVM... I was clicking the file name and github was sending me to an old version of the file. I see it now.